Makefile

```diff
@@ -164,9 +170,14 @@ build_tfhe_full: install_rs_build_toolchain
 		--features=$(TARGET_ARCH_FEATURE),boolean,shortint,integer -p tfhe --all-targets
 
 .PHONY: build_c_api # Build the C API for boolean and shortint
-build_c_api: install_rs_build_toolchain
+build_c_api: install_cbindgen
```

Comment:
I would add the install_rs_build_toolchain and install_rs_check_toolchain as dependencies as you refer to both in the Make target